app/src/main/java/com/reach5/identity/sdk/demo/JavaMainActivity.java

```diff
@@ -120,7 +120,11 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.menu_logout:
-                reach5.logoutWithProviders();
+                reach5.logout("",
```

Comment:
Why the empty string?

Reply:
I don't know what to put else, any idea?